README.md

```diff
@@ -111,6 +111,39 @@ for epoch in range(num_epochs):
     reader.reset() # Return to the beginning of dataset.
 ```
 
+### Reading Images as NumPy Arrays
+
+MLIO supports reading image files in jpeg, png, and recordio formats, using the [`ImageReader`](doc/python/data_reader.md#ImageReader) class. The code snippet below shows how you can quickly iterate through an image dataset in mini-batches.
```

Comment:
recordio is not an image format, but rather a way to find the boundaries of images in a stream of bytes. I would just say `MLIO supports reading image files in JPEG and PNG formats using ...`.